Modules/FramePool/QuestieFrame.lua

```diff
@@ -254,7 +254,7 @@ function _Qframe:UpdateTexture(texture)
     end
 
     self.texture:SetTexture(texture)
-    self.data.Icon = texture;
+    --self.data.Icon = texture;
```

Comment:
Not so sure what was going on here, from my reading of the code `texture` should always be `self.data.Icon` before the commit changing this anyway, so not sure why it was overwritten here...